kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/test/java/org/kie/workbench/common/screens/datasource/management/service/DriverDefEditorServiceTest.java

```
@@ -0,0 +1,169 @@
+        driverDef.setVersion( "version" );
+
+        driverDefEditorContent = new DriverDefEditorContent();
+        driverDefEditorContent.setDef( driverDef );
+        driverDefEditorContent.setProject( project );
+
+        originalDriverDef = new DriverDef();
+        originalDriverDef.setUuid( "uuid" );
+        originalDriverDef.setName( "driverNameOriginal" );
+        originalDriverDef.setDriverClass( TestDriver.class.getName() );
+        originalDriverDef.setGroupId( "groupIdOriginal" );
+        originalDriverDef.setArtifactId( "artifactIdOriginal" );
+        originalDriverDef.setVersion( "versionOriginal" );
+
+        try {
+            URL resource = getClass().getClassLoader().getResource( "DataSourceFiles" );
+            uri = resource.toURI();
+            when( artifactResolver.resolve( driverDef.getGroupId(), driverDef.getArtifactId(), driverDef.getVersion() ) ).thenReturn( uri );
+        } catch ( Exception e ) {
+        }
+    }
+
+    @Override
+    protected DefEditorContent getExpectedContent() {
+        return driverDefEditorContent;
+    }
+
+    @Override
+    protected String getExpectedDefString() {
+        return DriverDefSerializer.serialize( driverDef );
+    }
+
+    @Override
+    protected String getExpectedFileName() {
+        return driverDef.getName() + ".driver";
+    }
+
+    @Override
+    protected Def getExpectedDef() {
+        return driverDef;
+    }
+
+    @Override
+    protected Def getOriginalDef() {
+        return originalDriverDef;
+    }
+
+    @Override
+    protected String getOriginalDefString() {
+        return DriverDefSerializer.serialize( originalDriverDef );
+    }
+
+    @Override
+    protected void verifyCreateConditions( boolean global ) {
+        //we wants that:
+        try {
+            // 2) the definition was deployed.
+            verify( runtimeManager, times( 1 ) ).deployDriver( driverDef, DeploymentOptions.create() );
+        } catch ( Exception e ) {
```

Same comment about swallowing the exception.